docs/source/usersguide/output/statepoint.rst

```diff
@@ -4,7 +4,7 @@
 State Point File Format
 =======================
 
-The current revision of the statepoint file format is 15.
+The current revision of the statepoint file format is 16.
```

Comment:
I'm of the opinion that we only need to increment revisions at most once between releases.